php/meterpreter/ext_server_stdapi.php

```diff
@@ -1204,15 +1205,18 @@ function stdapi_registry_set_value($req, &$pkt) {
 register_command('stdapi_net_resolve_host', COMMAND_ID_STDAPI_NET_RESOLVE_HOST);
 function stdapi_net_resolve_host($req, &$pkt) {
     my_print("doing stdapi_net_resolve_host");
-    $hostname = packet_get_tlv($req, TLV_TYPE_HOST_NAME)['value'];
-    $family = packet_get_tlv($req, TLV_TYPE_ADDR_TYPE)['value'];
+    $hostname_tlv = packet_get_tlv($req, TLV_TYPE_HOST_NAME);
```

Comment:
Following the `_tlv` convention from elsewhere in the file

https://github.com/rapid7/metasploit-payloads/blob/4db3392680c5d7c875d6ce1ed747ab1fada996ac/php/meterpreter/ext_server_stdapi.php#L646-L647